rules/aws_cloudtrail_rules/aws_iam_compromised_key_quarantine.yml

```
@@ -0,0 +1,119 @@
+AnalysisType: rule
+LogTypes:
+  - AWS.CloudTrail
+Description: 'Detects when an IAM user has the AWSCompromisedKeyQuarantineV2 policy attached to their account.'
+DisplayName: "AWS Compromised IAM Key Quarantine"
+Enabled: true
+RuleID: "AWS.CloudTrail.IAMCompromisedKeyQuarantine"
+Filename: aws_iam_compromised_key_quarantine.py
+Severity: High
+Tags:
+  - AWS
+  - Configuration Required
+  - Identity and Access Management
+  - Persistence:Create Account
+Reports:
+  MITRE ATT&CK:
+    - TA0003:T1098
```

I think Valid Accounts: Cloud Accounts makes more sense here since no new account is being created.
```suggestion
    - TA0001:T1078.004
```